db/fixtures/miq_event_definition_sets.csv

```diff
@@ -8,6 +8,6 @@ vm_operational,VM Operation
 vm_configurational,VM Configuration
 vm_process,VM Lifecycle
 service_process,Service Lifecycle
-storage_operational, Datastore Operation
+storage_operational,Datastore Operation
```

Comment:
Oh, sorry I read this wrong. I thought you were *adding* the space.

This is fine :+1: